src/cljs/rems/administration/status_flags.cljs

```diff
@@ -11,46 +11,51 @@
    (or (get-localized-title item language)
        (:resid item)
        (:form/title item)
-       (:title item))))
+       (:title item)
+       (localized (:organization/name item)))))
 
 (defn- disable-button [item on-change]
   [:button.btn.btn-secondary.button-min-width
    {:type :button
-    :on-click #(on-change (assoc item :enabled false) [:span [text :t.administration/disable]
-                                                       " \"" [get-localized-title-for-anything item] "\""])}
+    :on-click #(on-change (assoc item :enabled false)
+                          [:span [text :t.administration/disable]
+                           " \"" [get-localized-title-for-anything item] "\""])}
    (text :t.administration/disable)])
 
 (defn- enable-button [item on-change]
   [:button.btn.btn-primary.button-min-width
    {:type :button
-    :on-click #(on-change (assoc item :enabled true) [:span [text :t.administration/enable]
-                                                      " \""  [get-localized-title-for-anything item] "\""])}
+    :on-click #(on-change (assoc item :enabled true)
+                          [:span [text :t.administration/enable]
+                           " \""  [get-localized-title-for-anything item] "\""])}
    (text :t.administration/enable)])
 
-; TODO consider naming enabled-toggle-button
+                                        ; TODO consider naming enabled-toggle-button
 (defn enabled-toggle [item on-change]
-  (if (:enabled item)
+  (if (or (:enabled item) (:enabled item))
```

Comment:
that... sounds a bit redundant :) leftover from the organization/enabled refactoring?

Reply:
Yeah I thought I got them all but missed some :(